omod/src/main/java/org/openmrs/module/idgen/web/controller/IdentifierSourceController.java

```diff
@@ -105,6 +103,12 @@ public void editIdentifierSource(ModelMap model, HttpServletRequest request,
 				}
 			}
 			model.addAttribute("otherCompatibleSources", otherCompatibleSources);
+			
+			List<String> prefixProviderBeanNames = new ArrayList<String>();
+			for (Class<? extends PrefixProvider> clazz : iss.getPrefixProviders()) {
```

Comment:
I would rather have a util method at the controller level that returns `prefixProviderBeanNames`.  I don't find it that relevant to have this added to the service layer itself.

You can get all the beans like this:
```java
Context.getRegisteredComponents(PrefixProvider.class)
```